packages/docs/docs/charting/index.md

```diff
@@ -14,6 +14,14 @@ Here is a **sample** of what a charting experience might look like - to be clear
 
 ![Chart sample](/img/tutorials/chart-sample.png)
 
+## Data Model
+
```

Comment:
Add some text here along the lines of: "All the elements you expect in a charting experience can be modeled in FHIR below are some examples of how the elements are represented